ansible/roles/fanout/tasks/sonic/fanout_sonic_incremental_202205.yml

```diff
@@ -0,0 +1,32 @@
+- name: collect fanout port config
+  port_config_gen:
+    hwsku: "{{ device_info[inventory_hostname]['HwSku'] }}"
+    hwsku_type: "{{ device_info[inventory_hostname]['HwSkuType'] | default('predefined') }}"
+    device_conn: "{{ device_conn[inventory_hostname] }}"
+  become: yes
+
+- name: build fanout incremental patch
+  template:
+    src: "sonic_deploy_incremental_202205.j2"
+    dest: "/tmp/config_patch.json"
+
+- name: apply patch to config db
+  shell: config apply-patch /tmp/config_patch.json
+  become: yes
+
+- name: disable all copp rules
```

Comment:
I remember copp rules cannot be applied without `config reload`. Can you check if this empty copp rule applied?

Reply:
I wasn't sure since it looks like a template file. I ask around and yes it needs a config reload. So its been added now.